web/src/js/views/settings.tsx

```diff
@@ -46,11 +46,11 @@ export default class Settings extends Component<{}, State> {
                 name: me.name,
             })
         })
-        Options.getOrigin().then(address => {
-            this.setState({
-                address: address,
-            })
-        })
+        // Options.getOrigin().then(address => {
```

Comment:
I'm not digging the commented out code, are we expecting to un-comment it in the near future? If not it should likely just be removed if we don't want it.

Reply:
That was my pre git coding coming through. I'll remove it